README.md

```diff
@@ -202,14 +203,18 @@ end
 
 ### Callbacks
 
-`#retriable` also provides a callback called `:on_retry` that will run after an exception is rescued. This callback provides the `exception` that was raised in the current try, the `try_number`, the `elapsed_time` for all tries so far, and the time in seconds of the `next_interval`. As these are specified in a `Proc`, unnecessary variables can be left out of the parameter list.
+`#retriable` also provides callbacks called `:on_retry`  and `:on_final_failure` that will run after an exception is rescued.  The `:on_retry` callback provides the `exception` that was raised in the current try, the `try_number`, the `elapsed_time` for all tries so far, and the time in seconds of the `next_interval`. The `:on_final_failure` callback provides everything but the `next_interval`. As these are specified in a `Proc`, unnecessary variables can be left out of the parameter list.
```

Comment:
"after an exception is rescued or fails, respectively"

Reply:
👍 . Updated but let me know if you were thinking of less than what I have now as well. Just want to make it clear that it doesn't pass `next_interval` to the `on_final_failure` proc as well